rulebooks/journald_events.yml

```diff
@@ -0,0 +1,16 @@
+---
+- name: Journald events
+  hosts: all
+  sources:
+    - name: Return severity 6 messages
```

Comment:
The meaning of name does match the `match`

Reply:
So that's good?